### ISSUE TITLE
Config Temperature HUD feature request

### ISSUE THREAD

**Issue by SQbuilder** [Glitchfiend/ToughAsNails#731]
## Feature Request

[ Config Temperature HUD feature request]</br>

### What feature are you suggesting?

[ Add a Config option to customize the transparency of the hot/frigid temperature overlays that appear over your screen. Aswell as the customizability of the size of the "oval hole" in the overlay. ]</br>

### Why should it be added?

[ Right now the overlay for being too warm / too cool is very intrusive in my opinion. Visibility is a must, especially within mod-packs that have several HUD elements that clutter the screen enough as it is. Thank you for consideration. ]</br>

**Comment by Forstride** [comment]
I'm gonna go out on a limb and say you're playing RLCraft, which has additional visual effects for TAN added by another mod, not us.

**Comment by Stapleton** [comment]
Im gonna bump this as its an actual issue to not be able to move the center circle or change the default temperature overlays in game. I am using TAN in my own pack and I very much have been looking for some solutions to hud problems and it seems like a lot of mods have issues with hud configurability. I can provide proof, but all it takes is to look at the client config files to find a lack of hud configurability in a couple spots.

At the very least, add a config option for the missing pieces. At the most, add FancyHUD/SpiffyHUD compatibility.

**Comment by xilenasu** [comment]
@SQbuilder not sure if still relevant, but it's Enhanced Visuals. If you look at the Enderman effect, aka Slender in config, it's the same shape, different texture.
It's weird that it changes the split-second that the body temperature from TAN changes...

tl;dr
disable temperature/slender effects in Enhanced Visuals config.
